modules/support-log-gather-configure-cli.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * support/support-log-gather-operator-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="support-log-gather-config-cli_{context}"]
+= Configuring a {support-log-gather} instance
+
+You must create a `MustGather` custom resource (CR) from the command-line interface (CLI) to automate the collection of diagnostic data from your cluster. This process also automatically uploads the data to a Red{nbsp}Hat Support case.
+
+:FeatureName: Support Log Gather
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have installed the oc CLI tool.
+
+* You have installed {support-log-gather} in your cluster.
+
+* You have a Red{nbsp}Hat Support case ID.
+
+* You have created a Kubernetes secret containing your Red Hat Customer Portal credentials. The secret must contain a username field and a password field.
+
+* You have created a service account.
+
+.Procedure
+
+. Create a YAML file for the `MustGather` CR, such as `support-log-gather.yaml`, that contains the following basic configuration::
++
+.Example `support-log-gather.yaml`
++
+[source, yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: MustGather
+metadata:
+  name: example-mg
+  namespace: must-gather-operator
+spec:
+  serviceAccountName: must-gather-operator
+  audit: true
+  proxyConfig:
+    httpProxy: "http://proxy.example.com:8080"
+    httpsProxy: "https://proxy.example.com:8443"
+    noProxy: ".example.com,localhost"
+  mustGatherTimeout: "1h30m9s"
+  uploadTarget:
+    type: SFTP
+    sftp:
+      caseID: "04230315"
+      caseManagementAccountSecretRef:
+        name: mustgather-creds
+      host: "sftp.access.redhat.com"
+  retainResourcesOnCompletion: true
+  storage:
+    type: PersistentVolume
+    persistentVolume:
+      claim:
+        name: mustgather-pvc
+      subPath: must-gather-bundles/case-04230315
+----
++
+For more information on the configuration parameters, see "Configuration parameters for {support-log-gather} custom resource".
+
+. Create the `MustGather` object by running the following command:
++
+[source, terminal]
+----
+$ oc create -f support-log-gather.yaml
+----
+
+.Verification
+
+. Verify that the `MustGather` CR was created by running the following command:
++
+[source, terminal]
+----
+$ oc get mustgather
+----
++
+.Example output
++
+[source, terminal]
```

Comment:
Some effort is underway to remove example codeblocks that do not add much value. Maybe a sentence such as the following code remove this small example output codeblock:

Example output shows the name of the CR, such as `example-mg` and its creation age in seconds.